ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -2113,3 +2120,108 @@ - (BOOL)isEqual:(id)object {
 }
 
 @end
+
+
+#pragma mark - ORKLocationAnswerFormat
+
+@implementation ORKLocationAnswerFormat
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        _useCurrentLocation = YES;
+    }
+    return self;
+}
+
+- (BOOL)isAnswerValidWithString:(nullable NSString *)text {
+    BOOL isValid = [super isAnswerValidWithString:text];
+    
+    MKPlacemark *placemark = [MKPlacemark ork_placemarkWithJSONString:text];
+    
+    if (placemark && CLLocationCoordinate2DIsValid(placemark.coordinate)) {
+        isValid = YES;
+    } else {
+        isValid = NO;
+    }
+    
+    return isValid;
+}
+
+- (NSString *)localizedInvalidValueStringWithAnswerString:(NSString *)text {
+    if (![text length]) {
+        return nil;
+    }
+    
+    MKPlacemark *placemark = [MKPlacemark ork_placemarkWithJSONString:text];
+    CLLocationCoordinate2D coordinate = placemark.location.coordinate;
+    NSMutableString *string = [NSMutableString new];
+    
+    if (coordinate.latitude < -90.0) {
+        [string appendString:ORKLocalizedString(@"LOCATION_ALERT_MESSAGE_LATITUDE_BELOW_MINIMUM", nil)];
```

Comment:
Error messages removed. Adding the ORKPlacemark class that handles validation removed the need for the validation error messages entirely.